packages/transformer-attributify-jsx/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import type { SourceCodeTransformer } from '@unocss/core'
+
+const elementRE = /<!--[\s\S]*?-->|<(\/?)([a-zA-Z][-.:0-9_a-zA-Z]*)((?:\s+[^>]*?(?:(?:'[^']*')|(?:"[^"]*"))?)*)\s*(\/?)>/gs
+
+export default function transformerAttributifyJsx(): SourceCodeTransformer {
+  return {
+    name: 'attributify-jsx',
+    enforce: 'pre',
+    transform(code) {
```

Comment:
We might need to check the extension and only apply for `.jsx` and `.tsx`

Reply:
yeah, and we should also apply for `.js` because you can use jsx in `.js` file.